### ISSUE TITLE
Update Charting Fallback

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#345]
The chart currently can't be read by screen readers. The chart should be represented as a table for screen readers.

**Comment by wilsuriel03** [comment]
This has been Resolved, coming up on the PR

**Comment by bbertucc** [comment]
@kevinandrews1 this is ready for your review

**Comment by kevinandrews1** [comment]
@wilsuriel03 @bbertucc We should probably have a heading to separate the chart data and the messages table since it semantically looks like the messages table is part of the chart data. A separate issue though so I"ll close this one out--it's good.
